addons/account/models/account_move.py

```diff
@@ -1494,6 +1492,13 @@ def _move_autocomplete_invoice_lines_write(self, vals):
             invoice.write(values)
         return True
 
+    @api.returns('self', lambda value: value.id)
+    def copy(self, default=None):
+        default = dict(default or {})
+        if not default.get('date') and self.company_id._get_user_fiscal_lock_date() > date(1970, 1, 1):
```

Comment:
> date.min

Reply:
No, because if the fiscal lock date is not set, the `date.min` will be returned by `_get_user_fiscal_lock_date`
This will be interpreted as `'1-1-1'` by the ORM, and it will not recognize it as a valid date.

Reply:
The check on the date was not correct though